### ISSUE TITLE
Update the README.md

### ISSUE THREAD

**Issue by hugolpz** [jsoma/tabletop#132]
Hello there,

The section [#Getting Started](https://github.com/jsoma/tabletop#1-getting-your-data-out-there) is **misleading**, giving incorrect information on which key or url to look for.

## Current README.md
It recommend : 

>  "In Google Docs, then go up to the File menu and pick Publish to the web. Fiddle with whatever you want, then click Start publishing. A URL will appear, something like https://docs.google.com/spreadsheet/pub?hl=en_US&hl=en_US&key=0AmYzu_s7QHsmdDNZUzRlYldnWTZCLXdrMXlYQzVxSFE&output=html
> 
> Copy that! In theory you're interested in the part between key= and & but you can use the whole thing if you want.

![screenshot from 2016-12-05 15-54-38](https://cloud.githubusercontent.com/assets/1420189/20889505/90b8dff8-bb03-11e6-957d-a49d4a34298f.png)

Which is **incorrect, misleading and [failing](http://codepen.io/anon/pen/LbdxvG).**

![screenshot from 2016-12-05 16-22-20](https://cloud.githubusercontent.com/assets/1420189/20890414/161c802a-bb07-11e6-86ff-02372277cbb3.png)

With the errors : 
```
Unexpected token I in JSON at position 0
Uncaught TypeError: Cannot read property 'feed' of undefined
```

## The issue
As of 2016/12 and for about 2 years at least, Google provides 2 groups of urls for a same Google sheet.

`File > Publish to the web` VS `Share` **provide differents urls**.

![screenshot from 2016-12-05 15-53-10](https://cloud.githubusercontent.com/assets/1420189/20890176/42225100-bb06-11e6-8cfb-2acbeaa829d4.png)
![screenshot from 2016-12-05 15-54-10](https://cloud.githubusercontent.com/assets/1420189/20890177/42228062-bb06-11e6-8095-25541dee7cac.png)

## Fix
Update the README.md, suggest to : 

1. ` File > publish to the web` : to make your file available
2. Click on `Share` button, share to the web AND pick the key from the url provided there.

![screenshot from 2016-12-05 16-20-38](https://cloud.githubusercontent.com/assets/1420189/20890347/d9efec22-bb06-11e6-858f-231c9ab959fa.png)

**Comment by jsoma** [comment]
I understand how that could be frustrating! Unfortunately I've spent the past hour publishing spreadsheets from a half-dozen Google accounts - personal, organizational, US, European - and for some reason *I just can't reproduce it*.

If you look at your "Publish to the web" url it has a `/d/e/` in it - all of the ones I can produce just have a `/d/` in them. I'm wondering if it has to do with your organizational account? **If you check the "Require viewers to sign in..." box, does it still have `/d/e/` in it?**

Either way, it's good to know that the "Share" url works even if the Publish to web one doesn't. I'll definitely make a note in the README about that.

**Comment by hugolpz** [comment]
Ok. Thanks for your answer.
There is something interesting. Following your answer, I went again to check out my spreadsheet, and, oh wonder.... 

![screenshot from 2016-12-09 12-58-20](https://cloud.githubusercontent.com/assets/1420189/21048388/53817672-be0f-11e6-8fa1-e1251c869211.png)

Content of `File > publish to the web`, the url, **has now changed to the right key**.

Note:
I made my former test **right after** publishing on the web. It is possible that the `File > publish to the web` url need to close **hard reload the page so the key is updated**.
